Makefile

```diff
@@ -61,11 +61,13 @@ ifneq ($(HAS_SERVER),)
 ifeq ($(MM_DEBUG),)
 	cd server && env GOOS=linux GOARCH=amd64 $(GO) build $(GO_BUILD_FLAGS) -o dist/plugin-linux-amd64;
 	cd server && env GOOS=darwin GOARCH=amd64 $(GO) build $(GO_BUILD_FLAGS) -o dist/plugin-darwin-amd64;
+	cd server && env GOOS=darwin GOARCH=arm64 $(GO) build $(GO_BUILD_FLAGS) -o dist/plugin-darwin-arm64.exe;
```

Comment:
Why is common windows file extension `.exe` used here?

Reply:
Ugh, I know now why I was still sitting on this -- this was just one commit from a two parter that I forgot to cherry-pick over here :) -- should be fixed now!